src/context/Context.tsx

```diff
@@ -1,18 +1,18 @@
 import React, { useState, useEffect, createContext } from 'react';
 
-import { fetchData } from '@/apis/fetchData';
+import { getItems } from '@/apis/getItems';
 import {
-  API_KEY,
   NUMBER_OF_ITEMS,
   CONTENT_TYPE,
   DEFAULT_SEARCH_VALUE,
   DELAY,
   MIN_SEARCH_CHARACTERS,
   QUERY_TYPE,
 } from '@/constants/constantValues';
-import { ContextProps, AppContextInterface, IShow, IMovie } from '@/types/types';
+import { ContextProps, AppContextInterface, AppContextActionsInterface, IShow, IMovie } from '@/types/types';
```

Comment:
No need for using the suffix **Interface** in TS types/interfaces. If you want to explicitly mark the TS type as an interface, then just use **I** as a prefix (similar to C#). But make sure that you have consistency at the project level.